x-pack/plugins/lens/public/visualizations/partition/visualization.test.ts

```diff
@@ -169,11 +169,11 @@ describe('pie_visualization', () => {
     describe('converting to legendStats', () => {
       it('loads a chart with `legendStats` property', () => {
         const persistedState = getExampleState();
-        persistedState.layers[0].legendStats = ['values' as LegendStats.values];
+        persistedState.layers[0].legendStats = ['value' as PartitionLegendValue.Value];
```

Comment:
Why not using directly?
```suggestion
        persistedState.layers[0].legendStats = [PartitionLegendValue.Value];
```

The same in the rest of the test